src/lib/utils/router.js

```diff
@@ -84,7 +84,12 @@ function onClick(e) {
   // nb. If this ever supports Shadow DOM, we can use .composedPath to find
   // the nearest link inside an open Shadow Root.
   const link = e.target.closest("a[href]");
-  if (!link || link.target || link.host !== location.host) {
+  if (
+    !link ||
+    link.target ||
+    link.host !== location.host ||
+    link.pathname.match(/\.(jpg|png|gif|svg)$/)
```

Comment:
Thanks, this is fine for now. We were just chatting about whether to disallow all "."-containing links, but we also (technically) allow you to load /article-name/index.html, so...